packages/widget/src/components/address-input/address-input.ts

```diff
@@ -20,11 +20,10 @@ export class AddressInput extends BaseComponent {
   address: string = '';
 
   @property({ attribute: false })
-  onAddressChange: (address: string) => void = () => {};
+  onAddressChange: (address: string, errorMessage?: string | null) => void =
+    () => {};
 
-  @property({
-    type: String
-  })
+  @property()
```

Comment:
```suggestion
  @property({ type: Object })
```

Reply:
I think it's not necessary as we are not using it as HTML attribute. I added { attribute: false } to indicate that we use it in code internally outside the render